sdk/containerregistry/Azure.Containers.ContainerRegistry/api/Azure.Containers.ContainerRegistry.netstandard2.0.cs

```diff
@@ -123,6 +123,14 @@ public enum ServiceVersion
             V1_0 = 1,
         }
     }
+    public static partial class ContainerRegistryModelFactory
+    {
+        public static Azure.Containers.ContainerRegistry.ArtifactManifestProperties ArtifactManifestProperties(string registryLoginServer = null, string repositoryName = null, string digest = null, long? size = default(long?), System.DateTimeOffset createdOn = default(System.DateTimeOffset), System.DateTimeOffset lastUpdatedOn = default(System.DateTimeOffset), Azure.Containers.ContainerRegistry.ArtifactArchitecture? architecture = default(Azure.Containers.ContainerRegistry.ArtifactArchitecture?), Azure.Containers.ContainerRegistry.ArtifactOperatingSystem? operatingSystem = default(Azure.Containers.ContainerRegistry.ArtifactOperatingSystem?), System.Collections.Generic.IReadOnlyList<Azure.Containers.ContainerRegistry.ArtifactManifestReference> manifestReferences = null, System.Collections.Generic.IReadOnlyList<string> tags = null, bool? canDelete = default(bool?), bool? canWrite = default(bool?), bool? canList = default(bool?), bool? canRead = default(bool?), string quarantineState = null, string quarantineDetails = null) { throw null; }
+        public static Azure.Containers.ContainerRegistry.ArtifactManifestReference ArtifactManifestReference(string digest = null, Azure.Containers.ContainerRegistry.ArtifactArchitecture architecture = default(Azure.Containers.ContainerRegistry.ArtifactArchitecture), Azure.Containers.ContainerRegistry.ArtifactOperatingSystem operatingSystem = default(Azure.Containers.ContainerRegistry.ArtifactOperatingSystem)) { throw null; }
```

Comment:
If we ever wanted to use EditorBrowsable Never here, would that get picked up on the creation methods automatically?

Reply:
If you apply `[EditorBrowsable(EditorBrowsableState.Never)]` and `[Browsable(false)]` to the hand-written partial part of the factory, it should affect all the methods in it.